services/core-api/app/api/now_applications/models/now_application.py

```diff
@@ -206,6 +254,12 @@ def remitted_permit(self):
             now_application_guid=self.now_application_guid,
             permit_amendment_status_code='RMT').one_or_none()
 
+    @hybrid_property
+    def related_permit_guid(self):
```

Comment:
Can we just call this `permit_guid`?

Reply:
discussed on a call and decided to keep as it is